### PR TITLE
support hcl in policygen

### LINKED ISSUE #185: Support HCL in policygen config.

### PR DESCRIPTION
**Pull request by xingao267**
Fixes #185

### CODE REVIEW

File: internal/policygen/load.go [hunk 1]
```diff
@@ -64,36 +69,75 @@ func loadConfig(path string) (*config, error) {
...
 	}
 
-	cj, err := yaml.YAMLToJSON(b)
-	if err != nil {
-		return nil, fmt.Errorf("convert config to JSON: %v", err)
+	// Save unmodified path to use in init().
+	originalPath := path
+
+	// Convert yaml to json so hcl decoder can parse it.
+	cj := b
```
Comment (umairidris) [inline internal/policygen/load.go:76]: Can't you continue using b?

Comment (xingao267) [reply internal/policygen/load.go]: Done

**Comment by xingao267** [comment]
@googlebot rescan

**Comment by xingao267** [comment]
@googlebot pls

**Comment by xingao267** [comment]
@googlebot rescan

**Comment by xingao267** [comment]
@googlebot rescan

**Comment by xingao267** [comment]
@googlebot rescan

**Pull request merged**